pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -577,7 +688,7 @@ func flushAllLogBundles(image string, iteration int, eveVersion string,
 var msgIDCounter = 1
 var iteration = 0
 
-func handleAppLogEvent(event logEntry, appLogs *logs.AppInstanceLogBundle) {
+func handleAppLogEvent(event logEntry, appLogs *logs.AppInstanceLogBundle) bool {
```

Comment:
Add comment about when it returns false (which is when the event is dropped)

Reply:
will do

Reply:
done